src/core/src/main/java/org/apache/jmeter/engine/DistributedRunner.java

```diff
@@ -262,21 +262,21 @@ private void println(String s) {
 
     private void errln(String s) {
         log.error(s);
-        stdErr.println(s);
+        stderr.println(s);
     }
 
     private void errln(String s, Exception e) {
         log.error(s, e);
-        stdErr.println(s + ": ");
-        e.printStackTrace(stdErr); // NOSONAR
+        stderr.println(s + ": ");
+        e.printStackTrace(stderr); // NOSONAR
     }
 
     public void setStdout(PrintStream stdout) {
         this.stdout = stdout;
     }
 
-    public void setStdErr(PrintStream stdErr) {
-        this.stdErr = stdErr;
+    public void setStderr(PrintStream stderr) {
```

Comment:
This looks like changing the public API. Would you please revert stdErr change?

Reply:
ok, i reverted